python/lsst/pipe/tasks/cli/cmd/commands.py

```diff
@@ -64,3 +68,15 @@ def make_discrete_skymap(*args, **kwargs):
 def register_skymap(*args, **kwargs):
     """Make a SkyMap and add it to a repository."""
     cli_handle_exception(script.registerSkymap.registerSkymap, *args, **kwargs)
+
+
+@click.command(cls=ButlerCommand,
+               short_help="Add subfilters for chaotic modeling.")
+@repo_argument(required=True)
+@num_subfilters_argument()
+@filter_names_argument()
```

Comment:
If these are band names and not filter names wouldn't it be less confusing to call this band_names_argument?